### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by mpujolWolox**
## Description
- Responsive design

## Screenshoots
image

## Trello Card
- https://trello.com/c/41ZYX0pf

### CODE REVIEW

File: scss/responsive.scss [hunk 1]
```diff
@@ -0,0 +1,56 @@
+@media only screen and (max-width: 1024px) {
```
Comment (mmamaniwolox) [inline scss/responsive.scss]: We generally dont use a separated file for responsive content.
You should do this media query inside each responsive scss file

File: scss/responsive.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  .flex-body {
+
```
Comment (mmamaniwolox) [inline scss/responsive.scss]: Delete this line

File: scss/responsive.scss [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    .train-header .dev{
+      display: none;
+    }
+
+    .train-main {
+
```
Comment (mmamaniwolox) [inline scss/responsive.scss]: Delete line

File: scss/responsive.scss [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        }
+      }
+    }
+
+    .train-footer {
+      grid-template-areas: 
+        "txt"
+        "logo"
+        "copyright";
+      grid-template-columns: 1fr;
```
Comment (mmamaniwolox) [inline scss/responsive.scss]: You dont need to specify columns when using template-areas

File: scss/responsive.scss [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+          background-color: transparent;
+          border-radius: 0;
+
+          &:first-child {
+            grid-column: auto;
+            grid-row: auto;
+            position: static;
+          }
+
+          &:not(:last-child) {
```
Comment (mmamaniwolox) [inline scss/responsive.scss]: This is quite complex, rather do `border-top` in tech and `border-top: 0` in `&:first-child`

File: scss/navbar.scss [hunk 6]
```diff
@@ -18,3 +18,10 @@
     font-size: $text-medium;
   }
 }
+
+@media only screen and (max-width: 1024px) {
+
+  .train-header .dev{
```
Comment (SKOLZ) [inline scss/navbar.scss]: add space before curly brace

**Pull request merged**